desc/objectives/_coils.py

```diff
@@ -730,6 +730,11 @@ class CoilSetMinDistance(_Objective):
         a large number of coils, or if the resolution is very high, setting this to a
         small value will reduce peak memory usage at the cost of slightly increased
         runtime.
+    use_centers : bool
+        Whether or not to only consider the coil centers, rather than the position
+        along the coil itself. This can reduce the computational cost in situations
+        where a coil's position is well-approximated by its center, for example
+        if the coils have a very small radius.
```

Comment:
may want to mention that in this case the min distance target should be larger than the radius for this objective to actually work correctly. May also want to mention dipole coils specifically since that's the common use case